src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/StandardContents/index.tsx

```diff
@@ -14,6 +14,12 @@ type Props = Pick<BlockProps, 'node'>
 const StandardContents: FC<Props> = ({ node }) => {
   const isWaiting = node.waitOnReceive
 
+  const getWidth = () => {
```

Comment:
I don't know if this issue is related to this piece of code but pretty much all my transitions are now aligned differently ... 

![Screen Shot 2021-03-12 at 8 47 20 AM](https://user-images.githubusercontent.com/955524/110948616-91468c80-830f-11eb-9465-04c1447ed5ab.png)